src/services/shopify-client.ts

```diff
@@ -62,9 +63,11 @@ export class HttpShopifyClient implements ShopifyClient {
         name: response.dopplerListName,
         amountSubscribers: response.importedCustomersCount,
         state:
-          response.syncProcessInProgress === true
+          !!response.syncProcessInProgress && response.syncProcessInProgress !== 'false'
```

Comment:
Mmm, I am not sure about this, it seems that the server is returning not well-formed data.

Let me think it a little more before merging.

Reply:
Oh, you have merged it just when I was commenting. 

Could you review if it will continue working fine when the backend result be ok?

Reply:
Please let me know @andresmoschini when it's done.

Reply:
I've tested this and whether it's boolean or text it will work just fine.

Reply:
Great!